src/api/transaction.rs

```diff
@@ -575,18 +884,19 @@ fn contract_update_operations(
                         "{}{}_{}",
                         ACCOUNT_CONTRACT_PREFIX, from.index.index, from.subindex.sub_index
                     ),
-                    Some(amount_from_uccd(-(amount.microgtu as i128))),
+                    Some(amount_from_uccd(-(amount.microccd as i128))),
                     None,
                 ));
                 ops.push(account_transaction_operation::<Value>(
                     next_index + 1,
                     details,
                     to.to_string(),
-                    Some(amount_from_uccd(amount.microgtu as i128)),
+                    Some(amount_from_uccd(amount.microccd as i128)),
                     None,
                 ));
                 next_index += 2;
             }
+            _ => {}
```

Comment:
I would recommend listing Interrupted and Resumed here explicitly in order that we are informed later on that new cases need to be handled, if new contract events are going to be added.

Reply:
Added
```
            ContractTraceElement::Interrupted { .. } => {}
            ContractTraceElement::Resumed { .. } => {}
```
Is that what you mean?

Reply:
Yes.